src/RMPI/onesided.jl

```diff
@@ -113,10 +112,11 @@ Deposit a vector `buf` into the MPI window `s` on rank `targetrank`. If
 @inline function put(buf::Vector{T}, len, targetrank, s::MPIOneSided{T}) where T
     @boundscheck len ≤ length(buf) && len ≤ s.capacity ||
         error("Not enough space left in buffer")
-    b_buffer = MPI.Buffer(buf, len, s.DT_b)
-    MPI.Put(b_buffer, targetrank, 0, s.b_win)
-    l_buffer = MPI.Buffer([len,], 1, s.DT_l)
-    MPI.Put(l_buffer, targetrank, 0, s.l_win)
+    # TODO: using Buffers onlt works on master MPI.jl
+    #b_buffer = MPI.Buffer(buf, len, s.DT_b)
+    MPI.Put(buf, targetrank, s.b_win)
+    #l_buffer = MPI.Buffer([len,], 1, s.DT_l)
+    MPI.Put([len], targetrank, s.l_win)
```

Comment:
I do not understand the reason for this change. Is it safe?
We went to using `MPI.Buffer` from the master branch because of linear memory growth and crashes. MPI.jl master had been fixed for us by Simon because of that reason. The current code looks like the old code. Has the original problem been fixed in some other way by now?